### ISSUE TITLE
Information dashboard

### ISSUE THREAD

**Issue by DWGodwin** [DWGodwin/glooper#19]
A collapsable dashboard of project-level information should be present in the interface. Example information:
- Number of positive samples in training/validation/test dataset
- Performance of previous training runs on test/validation set (could be delegated to a TensorFlow dashboard for familiarity)
- Information about dataset representativeness (things like pixel histograms/embedding analysis, especially comparing training and validation)
- Time per label
- Total project time
- User settings such as desired accuracy
This list is not exhaustive, but should include things the user should always be able to access from the frontend. There could also be a config for this, so a project manager can turn these displays on/off as desired
